src/xbps/index.md

```diff
@@ -74,6 +74,13 @@ kernels](../config/kernel.md#removing-old-kernels)" for further information.
 
 ## Finding Files and Packages
 
+To list all the files provided by an installed package, use
+[xbps-query(1)](https://man.voidlinux.org/xbps-query.1):
```

Comment:
Perhaps add something like "If you can't find a file or program you expected to find when installing a package, you can list all the files provided ..." ?

Reply:
Good idea - PR updated.